ushahidi/platform-client-mzima
# [Outreachy Task Submission]: Fix missing tick marks for collections with existing posts

### Description:
This pull request fixes the issue: ushahidi/platform#4863 where tick marks are not displayed for collections containing posts in the "Add to Collection" popup within the post view page. By implementing this fix, we ensure that users can easily identify collections with existing posts, improving clarity and user experience.

### Testing Checklist:
1. Launch the mobile application with `npm run mobile:serve`.
2. Select a deployment or create a new one.
3. Log in to the application.
4. Navigate to a post.
5. Within the post view, click on the "add_to_collection" icon.
6. Verify that tick marks are displayed for collections with existing posts.

https://github.com/ushahidi/platform-client-mzima/assets/68381641/89e5b2a3-493a-4349-a94b-6fe44746a91e
